operator/v1/types_csi_cluster_driver.go

```diff
@@ -40,6 +40,18 @@ type ClusterCSIDriver struct {
 // CSIDriverName is the name of the CSI driver
 type CSIDriverName string
 
+// +kubebuilder:validation:Pattern=`^(Managed|Unmanaged|Removed)$`
+type StorageClassStateName string
+
+const (
+	// ManagedStorageClass means that the operator is actively managing its storage classes.
+	ManagedStorageClass StorageClassStateName = "Managed"
+	// UnmanagedStorageClass means that the operator is not actively managing storage classes.
```

Comment:
As above would like this expanded, what does being unmanaged actually mean for an end user